src/consumption.cpp

```diff
@@ -207,6 +213,106 @@ nutrients player::compute_effective_nutrients( const item &comest ) const
     }
 }
 
+// Calculate range of nutrients obtainable for a given item when crafted via
+// the given recipe
+std::pair<nutrients, nutrients> player::compute_nutrient_range( const item &comest,
+        const recipe_id &recipe_i ) const
+{
+    if( !comest.is_comestible() ) {
+        return {};
+    }
+
+    // if item has components, will derive calories from that instead.
+    if( comest.has_flag( "NUTRIENT_OVERRIDE" ) ) {
+        nutrients result = compute_default_effective_nutrients( comest, *this );
+        return { result, result };
+    } else {
```

Comment:
```suggestion
    }
```
And remove the indentation of the remaining code.

Reply:
Thanks.  Done.